src/app/components/publiclists/teamlist/teamcard/teamcard.component.ts

```diff
@@ -156,17 +203,26 @@ export class TeamcardComponent implements OnInit, OnChanges {
     }
     const temp = this.team['members'];
     let memberString = '';
+    let memberIdString = '';
     for (let i = 0; i < temp.length; i++) {
       if (temp[i]['member_name']) {
         memberString = memberString + ', ' + temp[i]['member_name'];
+        memberIdString = memberIdString + ', ' + temp[i]['id'] ;
       } else {
         memberString = memberString + ', ' + temp[i]['user'];
+        memberIdString = memberIdString + ', ' + temp[i]['id'] ;
       }
     }
     if (memberString !== '') {
       memberString = memberString.slice(2, memberString.length);
     }
+    if (memberIdString !== '') {
+      memberIdString = memberIdString.slice(2, memberIdString.length);
+    }
     this.teamView['members'] = memberString;
+    this.teamView['member_ids'] = memberIdString;
+    this.memberArray = memberString.split(',');
+    this.memberIdArray = memberIdString.split(', ');
```

Comment:
Can we change the logic of doing like this? I think at the end we need an array with member and memberId array so how about directly initializing an array and push the value to it rather than initializing an string and then appening `,`(comma) and then value @Kajol-Kumari What do you think?

Reply:
> Can we change the logic of doing like this? I think at the end we need an array with member and memberId array so how about directly initializing an array and push the value to it rather than initializing an string and then appening `,`(comma) and then value @Kajol-Kumari What do you think?

@Sanji515 I am doing it this way as i need to store the members in the form of `,` separated string in the `teamView` as it is being used to display the list of member. So, i need both comma separated string as well as array and so i am doing it this way.

Reply:
But now I think you are displaying as a list with two columns one with the member username and another with delete icon. I don't see any comma separated text in UI.